audio_to_multiple_pose_gan/my_audio_to_pose_net/my_first_model.py

```diff
@@ -242,21 +242,33 @@ def preprocess_dataset(files):
 # compile, aka define the optimizer and loss
 model.compile(
     optimizer=tf.keras.optimizers.Adam(),
-    loss=tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True),
+    loss=tf.keras.losses.CategoricalCrossentropy(from_logits=True),
     metrics=['accuracy'],
 )
 
+
+labs_categories = {"up": 0, "left": 1, "right": 2, "down": 3, "no": 4, "yes": 5, "go": 6, "stop": 7}
+def categorize(command: str):
+    v = np.zeros([num_labels])
+    v[labs_categories[command]] = 1.0
+    return v
+
+labs_vec = np.array([categorize(s) for s in labs])
```

Comment:
I had to convert the string labels into numbers for a loss function to make sense. I chose the one-hot encoding.